.github/dependabot.yml

```diff
@@ -35,18 +35,23 @@ updates:
       - dependency-name: github.com/elastic/go-seccomp-bpf
       - dependency-name: github.com/elastic/toutoumomoma
       - dependency-name: github.com/elastic/ebpfevents
+      # Team:Team:obs-ds-hosted-services, Team:obs-infraobs-integrations
+      # Azure SDK dependencies
+      - dependency-name: github.com/Azure/azure-sdk-for-go/*
+      - dependency-name: github.com/Azure/azure-event-hubs-go/*
+      - dependency-name: github.com/Azure/azure-storage-blob-go/*
+      - dependency-name: github.com/Azure/go-autorest/*
+      # GCP SDK dependencies
+      - dependency-name: cloud.google.com/go/*
     groups:
       # Cloud providers' SDK dependencies
       azure-sdks:
         patterns:
-          - "github.com/Azure/azure-sdk-for-go/*"
-          - "github.com/Azure/azure-event-hubs-go/*"
+          - "github.com/Azure/azure-*-go"
           - "github.com/Azure/go-autorest/*"
-          - "github.com/Azure/azure-storage-blob-go/*"
       gcp-sdks:
         patterns:
           - "cloud.google.com/go/*"
-          - "google.golang.org/*"
```

Comment:
Just double checking: this is on purpose?

The namespace at least seems to exist https://pkg.go.dev/google.golang.org/api

Reply:
The `cloud.google.com/go/*` packages focus on Google Cloud Platform (GCP) specifically, allowing the use of APIs for services like Compute Engine, Cloud Storage, BigQuery, Pub/Sub, and others.

The `https://pkg.go.dev/google.golang.org/api` packages are applicable instead for interacting with various Google APIs, such as Google Drive, Google Calendar, YouTube, and others.

For this Dependabot configuration, I want to focus only on GCP. My bad for not adding a comment in the PR description; I updated it accordingly).

Reply:
@dliappis, let me know what you think 🙇

Reply:
@zmoog Thanks for the explanation. I will approve the PR on behalf of @dliappis since he isn't available this week.

Reply:
Thank you, @rowlandgeoff!